com.unity.render-pipelines.high-definition/Editor/RenderPipeline/Camera/HDCameraUI.Rendering.Drawers.cs

```diff
@@ -6,9 +6,9 @@ namespace UnityEditor.Rendering.HighDefinition
 {
     using CED = CoreEditorDrawer<SerializedHDCamera>;
 
-    static partial class HDCameraUI
+    internal static partial class HDCameraUI
```

Comment:
internal is default visibility for class outside struct / class. This is not needed

Reply:
thanks!